standard/classes.md

```diff
@@ -443,7 +443,7 @@ The nullability of the type argument need not match the nullability of the type
 > *Note*: To specify that a type argument is a nullable reference type, don’t add the nullable type annotation as a constraint (use `T : class` or `T : BaseClass`), but use `T?` throughout the generic declaration to indicate the corresponding nullable reference type for the type argument. *end note*
 
 <!-- Remove in C# 9, when this is allowed -->
-The nullable type annotation, `?`, can’t be used on an unconstrained type argument.
```

Comment:
Code that was erroneously allowed by the original sentence, while it should have been disallowed (it's not unconstrained, but it's still not allowed to use `?`):
```cs
#nullable enable

class C
{
    void M<T>(T? test) where T : System.IDisposable
    {
    }
}
```